scripts/topology/fourier_to_ryckaert_bellemans.py

```diff
@@ -29,19 +29,42 @@
 # (2019)
 
 
+import argparse
 import numpy as np
 
-f = np.array([V1, V2, V3, V4])  # Enter here the four Fourier dihedral coefficients
-if len(f) != 4:
-    raise ValueError("f must have length 4")
+def calc_rb_coeff(f):
+    if len(f) != 4:
+        raise ValueError("f must have length 4")
 
-rb = np.zeros(6)
-rb[0] = f[1] + 0.5 * (f[0] + f[2])
-rb[1] = 0.5 * (-f[0] + 3*f[2])
-rb[2] = -f[1] + 4*f[3]
-rb[3] = -2*f[2]
-rb[4] = -4*f[3]
-rb[5] = 0
+    rb = np.zeros(6)
+    rb[0] = f[1] + 0.5 * (f[0] + f[2])
+    rb[1] = 0.5 * (-f[0] + 3*f[2])
+    rb[2] = -f[1] + 4*f[3]
+    rb[3] = -2*f[2]
+    rb[4] = -4*f[3]
+    rb[5] = 0
 
-print("The Ryckaert-Bellemans dihedral coefficients are")
-print("   ".join([str(i) for i in rb]))
+    return rb
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description = (
+            "Convert Fourier dihedral coefficients"
+            "  to Ryckaert-Bellemans dihedral coefficients."
+        )
+    )
+    parser.add_argument(
+        '-v',
+        dest='V',
+        type=float,
+        nargs=4,
+        required=True,
+        help="The four Fourier dihedral coefficients"
+    )
+    args = parser.parse_args()
+
+    # Enter here the four Fourier dihedral coefficients
```

Comment:
I find this comment confusing. Actually you are entering the coefficients via the command line and not in the script itself. Is there some special reason you put this comment here?

Reply:
No particular reason at all, I think it was there before so I left it in. Will remove with force push after you're done reviewing.